<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cboard_single</class>
 <widget class="QWidget" name="Cboard_single">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_menu_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>700</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_game">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>760</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>510</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="timetitle">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;当前时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="runtimelabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;00:00:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_time">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <zorder>runtimelabel</zorder>
  <zorder>back_menu_button</zorder>
  <zorder>quit_game</zorder>
  <zorder>start_button</zorder>
  <zorder>pause_button</zorder>
  <zorder>timetitle</zorder>
  <zorder>lcd_time</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
